### PR TITLE
Update styles

### PR DESCRIPTION
**Pull request by daniel-sargeant**
---
PR Title: [SUB-7633] \<Consume Component Library\>
---
### Description (optional)
Add updated stylesheets to component-library so frontend looks normal when consuming component-library.

### Changes
* Add front-end changes to stylesheets to component-library
* Update button component with recent changes on frontend.

[SUB-7633]: https://submittable.atlassian.net/browse/SUB-7633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/styles/variables.module.less [hunk 1]
```diff
@@ -91,7 +91,20 @@
...
 @navbar-height: 56px;
 
+@max-window-width: 1136px;
+
 @background-overlay: #3333334d;
 
 @desktop: ~'only screen and (min-width: 960px) and (max-width: 1199px)';
 @mobile: ~'only screen and (max-width: 959px)';
+
+@grid-max-columns: 12;
```
Comment (chan-fai-leung) [inline src/styles/variables.module.less:101]: do we want to export the `variables.module.less` + `mixins.module.less` so this file would be the single source of truth?

Comment (chan-fai-leung) [reply src/styles/variables.module.less:101]: pull FE main, there are new changes that is added to this file

**Comment by chan-fai-leung** [comment]
Code LGTM but minor questions

File: src/styles/variables.module.less (hunk 1 above)
Comment (daniel-sargeant) [reply src/styles/variables.module.less:101]: Good catch, updated.

How would we add the less modules as exports in the package? Would the webpack config need any changes?

Comment (chan-fai-leung) [reply src/styles/variables.module.less:101]: lets create a story for it. i would assume we just need to update the bundler to include the designated less file and expose it

**Review by chan-fai-leung — APPROVED** [review]

File: src/styles/variables.module.less (hunk 1 above)
Comment (daniel-sargeant) [reply src/styles/variables.module.less:101]: https://submittable.atlassian.net/browse/SUB-7869

**Pull request merged**